cdap-gateway/src/main/java/io/cdap/cdap/gateway/router/NettyRouter.java

```diff
@@ -89,16 +90,19 @@
   private final TokenValidator tokenValidator;
   private final UserIdentityExtractor userIdentityExtractor;
   private final boolean sslEnabled;
+  private final Cipher cipher;
   private InetSocketAddress boundAddress;
 
-  private DiscoveryServiceClient discoveryServiceClient;
+  private final DiscoveryServiceClient discoveryServiceClient;
   private Cancellable serverCancellable;
 
+
   @Inject
```

Comment:
I don't see any guice binding for the `Cipher` interface, is this class not instantiated through guice? If guice is not used, better remove the `@Inject` annotation

Reply:
Removed the annotation